imports/plugins/core/accounts/server/no-meteor/mutations/removeAccountFromGroup.js

```diff
@@ -0,0 +1,77 @@
+import SimpleSchema from "simpl-schema";
+import ReactionError from "@reactioncommerce/reaction-error";
+import ensureRoles from "/imports/plugins/core/accounts/server/no-meteor/util/ensureRoles";
+
+const inputSchema = new SimpleSchema({
+  accountId: String,
+  groupId: String
+});
+
+/**
+ * @name accounts/removeAccountFromGroup
+ * @memberof Mutations/Accounts
+ * @summary Removes a user from a group for a shop, and adds them to the default customer group.
+ * Updates the user's permission list to reflect.
+ * (NB: At this time, a user only belongs to only one group per shop)
+ * @param {Object} context - GraphQL execution context
+ * @param {Object} input - Necessary input for mutation. See SimpleSchema.
+ * @param {String} input.accountId - decoded ID of account on which entry should be updated, for admin
+ * @param {String} input.groupId - decoded ID of the group to remove account from
+ * @returns {Promise<Object>} with modified group
+ */
+export default async function removeAccountFromGroup(context, input) {
+  inputSchema.validate(input);
+  const { appEvents, collections, userHasPermission, userId: userIdFromContext } = context;
+  const { Accounts, Groups, users } = collections;
+  const {
+    accountId: providedAccountId,
+    groupId
+  } = input;
+
+  const { shopId } = await Groups.findOne({ _id: groupId }) || {};
```

Comment:
If no group is found, then `shopId` would be undefined which I think would cause either an error or unexpected results from the `userHasPermission` call below. It would be better to do an explicit check before the permission check.

```js
if (!group) throw new ReactionError("not-found", "Group not found");
```